Je viens de tester ton patch et publication d'une version 2.3.2 sur test.pypi.org, mais l'installation se passe mal :
```
$ python3.5 -m venv venv
$ venv/bin/pip install -i https://test.pypi.org/simple/ lib-Partage-BSS-univ-rennes==2.3.2
$ venv/bin/cli-bss.py --help
		Traceback (most recent call last):
		  File "venv/bin/cli-bss.py", line 9, in <module>
			from lib_Partage_BSS.exceptions import *
		ImportError: No module named 'lib_Partage_BSS'
$ find . -name lib_Partage_BSS
```

J'ai l'impression qu'avec ta modif, la bibliothèque n'est plus installée...

--- a/setup.py
+++ b/setup.py
@@ -12,7 +12,8 @@ setuptools.setup(
     long_description=long_description,
     long_description_content_type="text/markdown",
     url='https://github.com/dsi-univ-rennes1/libPythonBssApi',
-    packages=setuptools.find_packages(include=['xmljson', 'requests', 'wheel']),
+    packages=setuptools.find_packages(),
+    install_requires=['xmljson', 'requests', 'wheel'],
     classifiers=[
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: Apache Software License",

